server-db/src/main/resources/db/Dockerfile

```diff
@@ -1,5 +1,7 @@
-# this file is used to build a docker image for upgrading the database in RIA infra
-FROM nexus.riaint.ee:8500/liquibase/liquibase
+# by default use docker.io as Docker registry, overwrite with
+# --build-arg LIQUIBASE_IMAGE=custom.registry.io:8500/liquibase/liquibase
+ARG LIQUIBASE_IMAGE=docker.io/liquibase/liquibase:4.29.2
```

Comment:
Sonarqube is complaining about running docker as root. Maybe we can use `USER liquibase` here?

Reply:
Resolved with https://github.com/open-eid/cdoc2-capsule-server/pull/8/commits/d0d419bed0cc31065fb7e510dba0203ca388c152